Update ClickPipes schema registry options (#2489)
ClickHouse/clickhouse-docs: docs/en/integrations/data-ingestion/clickpipes/kafka.md

  ![Fill out connection details](./images/cp_step2.png)

5. Configure the schema registry. A valid schema is required for Avro streams and optional for JSON. This schema will be used to parse [AvroConfluent](../../../interfaces/formats.md/#data-format-avro-confluent) or validate JSON messages on the selected topic.

> Avro messages that can not be parsed or JSON messages that fail validation will generate an error.

Reviewer: ```suggestion
- Avro messages that cannot be parsed or JSON messages that fail validation will generate an error.
```
